docs/ingestion/supervisor.md

```diff
@@ -24,30 +24,40 @@ sidebar_label: Supervisor
   -->
 
 A supervisor manages streaming ingestion from external streaming sources into Apache Druid.
-Supervisors oversee the state of indexing tasks to coordinate handoffs, manage failures, and ensure that the scalability and replication requirements are maintained.
 
-This topic uses the Apache Kafka term offset to refer to the identifier for records in a partition. If you are using Amazon Kinesis, the equivalent is sequence number.
+Supervisors oversee the state of indexing tasks to coordinate [handoffs](../design/storage#indexing-and-handoff), manage failures, and ensure that the scalability and replication requirements are maintained.
 
-## Supervisor spec
+:::info
+This topic uses the Apache Kafka term _offset_ to refer to the identifier for records in a partition. If you are using Amazon Kinesis, the equivalent is _sequence number_.
+:::
 
-Druid uses a JSON specification, often referred to as the supervisor spec, to define streaming ingestion tasks.
-The supervisor spec specifies how Druid should consume, process, and index streaming data.
+Druid uses a JSON document, often referred to as a supervisor spec, to specify how Druid should consume, process, and index streaming data. A supervisor spec is a type of [native ingestion spec](../ingestion/ingestion-spec.md).
```

Comment:
Why did you choose to replace "JSON specification" with "JSON document"?

Reply:
```suggestion
Druid uses a JSON formatted supervisor spec, to specify how Druid should consume, process, and index streaming data. A supervisor spec is a type of [native ingestion spec](../ingestion/ingestion-spec.md).
```
It is not "often referred to" but always a "supervisor spec". JSON document reads vaguely here.